larq_compute_engine/tflite/kernels/bconv2d.cc

```diff
@@ -199,6 +199,14 @@ TfLiteStatus Prepare(KernelType kernel_type,
   TF_LITE_ENSURE_EQ(context, NumDimensions(post_activation_multiplier), 1);
   TF_LITE_ENSURE_EQ(context, NumDimensions(post_activation_bias), 1);
 
+  // Read the input dimensions. TF Lite has the same input format as TensorFlow:
+  // (B, H, W, Ci).
+  conv_params->batch = input->dims->data[0];
+  conv_params->input_height = input->dims->data[1];
+  conv_params->input_width = input->dims->data[2];
+  if (!conv_params->read_bitpacked_input)
+    conv_params->channels_in = input->dims->data[3];
```

Comment:
Just personal preference, feel free to ignore (I feel like I am betraying Python with this suggestion)
```suggestion
  if (!conv_params->read_bitpacked_input) {
    conv_params->channels_in = input->dims->data[3];
  }
```